src/ripple/protocol/LedgerFormats.h

```diff
@@ -232,6 +238,7 @@ enum LedgerSpecificFlags {
     lsfDefaultRipple =
         0x00800000,               // True, trust lines allow rippling by default
     lsfDepositAuth = 0x01000000,  // True, all deposits require authorization
+    lsfAMM = 0x02000000,          // True, AMM account
```

Comment:
I know it's specified in the spec, but this value conflicts with `lsfTshCollect`, which is reserved for hooks, and already in `develop`, though it's commented out. One or the other will have to change.